src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo_link" href="#home">
+        <img src="./images/logo.png"
+        alt="logo"
+        class="logo">
```

Comment:
![Screenshot 2023-01-25 at 15 45 55](https://user-images.githubusercontent.com/84275496/214683798-81eece9d-c5cc-4a8e-bd50-8101ea50c95e.png)

Reply:
Not from this task checklist but still